src/index.html

```diff
@@ -9,6 +9,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header-moyo">
+      <a href="#" class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+          class="logo__image"
+        >
+      </a>
+      <nav class="nav-moyo">
+        <ul class="nav-list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item" data-qa="hover">
+            <a href="#" class="nav__link active">НОУТБУКИ И КОМПЬЮТЕРЫ</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Гаджеты</a>
```

Comment:
```suggestion
            <a href="#" class="nav__link active">НОУТБУКИ И КОМПЬЮТЕРЫ</a>
          </li>
          <li class="nav__item">
            <a href="#" class="nav__link">Гаджеты</a>
```
Чомусь в одних випадках у тебе написано великими літерами, а в інших - маленькими. Все повинно бути в одному стиді